service/worker/deployment/deployment_workflow.go

```diff
@@ -51,31 +49,27 @@ type (
 	}
 )
 
-var (
-	defaultActivityOptions = workflow.ActivityOptions{
-		ScheduleToCloseTimeout: 1 * time.Hour,
-		StartToCloseTimeout:    30 * time.Second,
-		RetryPolicy: &temporal.RetryPolicy{
-			InitialInterval: 1 * time.Second,
-			MaximumInterval: 60 * time.Second,
-		},
-	}
-)
-
 const (
-	UpdateDeploymentSignalName        = "update_deployment"
-	UpdateDeploymentBuildIDSignalName = "update_deployment_build_id"
+	// Updates
+	RegisterWorkerInDeployment = "register-worker"
```

Comment:
Actually thinking more about this, `register-task-queue-worker` is more clear.

Reply:
no objection, agreed.